build-gcc-configuration-files/README

```diff
@@ -0,0 +1,25 @@
+Instructions to build gcc manually (no omnibus package)
+======================================================
+
+You will need gmp, mpfr and mpc to build gcc and then tell gcc where to find it
+building the omnibus package will do this for you (remembers that this will build
+gcc too but we are looking for developing and testing gcc in a separate tree):
+
+  git clone https://github.com/CodethinkLabs/omnibus-codethink-toolchain.git
+  cd omnibus-codethink-toolchain
+  omnibus build codethink-toolchain --override base_dir:./local workers:8
```

Comment:
It might be useful to provide which user should I log in and execute those 3 lines.

Reply:
That probably won't belong here. But in the other README.